static/src/javascripts/projects/common/modules/identity/autosignin.js

```diff
@@ -79,7 +79,7 @@ function (
         this.welcome = function (name) {
             var msg = '<p class="site-message__message">' +
                           'Welcome ' + name + ', you’re signed into the Guardian using Facebook, or ' +
-                          '<a href="' + config.page.idUrl + '/signout"/>sign out</a>.' +
+                          '<a data-link-name="fb-auto-signout" href="' + config.page.idUrl + '/signout"/>sign out</a>.' +
```

Comment:
Hey, I've just been told the signout events across the rest of the site say 'sign out' 

Can we please update the name to to be "fb auto | Sign out"